src/cmd-meta

```diff
@@ -36,34 +73,54 @@ def new_cli():
     schema = args.schema
     if args.skip_validation:
         schema = None
-    meta = Meta(args.workdir, args.build, schema=schema)
-
-    # support the coreos-assembler.* keys
-    def pather(val):
-        path = val.split('.')
-        if val.startswith("coreos-assembler."):
-            new_path = [f"{path[0]}.{path[1]}"]
-            new_path.extend(path[2:])
-            return new_path
-        return path
+    meta = GenericBuildMeta(args.workdir, args.build, schema=schema)
+    old_ts = meta.get(COSA_VER_STAMP)
 
     # Get keys
-    if args.get is not None:
-        dotpath = args.get[0]
+    if args.get is not None or args.get_value is not None:
+        dotpath = args.get[0] if args.get else args.get_value[0]
```

Comment:
If you prefer safe getters you can use `next(iter(listVar), defaultValue)` (note: if you don't define a default value and the entry is not found it will throw a [`StopIteration` error](https://docs.python.org/3/library/functions.html#next)), e.x.:
```python
dotpath = next(iter(args.get), next(iter(args.get_value), None))
```

Reply:
Its "safe" since we know one is at least not `None`. But also:
```
Traceback (most recent call last):
  File "/usr/lib/coreos-assembler/cmd-meta", line 127, in <module>
    new_cli()
  File "/usr/lib/coreos-assembler/cmd-meta", line 75, in new_cli
    dotpath = next(iter(args.get, []), next(iter(args.get_value), None))
TypeError: iter(v, w): v must be callable
```

Reply:
Ah yeah, they default to `None`.